### PR TITLE
es6-literals

### PR DESCRIPTION
**Pull request by kbao24**
es6-template-literals-2

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = 'Irvine Center Drive';
```
Comment (robgmerrill) [inline es6-template-literals/index.html:12]: string interpolation should have been done here

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      const address = number + ' ' + street;
+
+      console.log(`LearningFuze: ${number} ${street}`);
+
+      const firstName = 'Uzair';
+      const lastName = 'Ashraf';
+      const age = 25;
+
+      const bio =
```
Comment (robgmerrill) [inline es6-template-literals/index.html:21]: string interpolation should have been done here

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        ' years old.';
+
+      console.log(
+        `Bio: My name is ${firstName} ${lastName} and I am ${age} years old.`
+      );
+
+      const fruit = 'apple';
+      const item = 'pen';
+
+      const applePen = fruit + item;
```
Comment (robgmerrill) [inline es6-template-literals/index.html:37]: string interpolation should have been done here

**Review by robgmerrill — CHANGES_REQUESTED** [review]
String interpolation should have been done wherever there was string concatenation.

File: es6-template-literals/index.html (hunk 2 above)
Comment (kbao24) [reply es6-template-literals/index.html]: Got it, thanks!

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**